azure/Kqlmagic/kql_magic.py

```diff
@@ -194,6 +197,19 @@ def validate_cloud(self, cloud):
             )
 
 
+    @validate("login_code_destination")
+    def _valid_value_login_code_destination(self, proposal):
+        dest = proposal["value"]
+        if dest != "browser":
+            if not self.check_mailto(dest):
+                message = "The 'login_code_destination' trait of a {0} instance {1}".format(Constants.MAGIC_CLASS_NAME, str(e))
```

Comment:
e is not set. the code you copy paste is a try/catch and the catch containe the exception in e. you can do the same if check_mailto will throw exception when it fails

Reply:
yes I did that this morning..somehow got lost. fixed it now